hot-desk/src/main/java/com/exadel/sandbox/booking/dto/BookingResponseDto.java

```diff
@@ -14,8 +14,6 @@
 @JsonPropertyOrder({"id", "employeeId", "seatId", "parkingSpotId", "dates"})
 public class BookingResponseDto extends BookingBaseDto {
     private Long id;
-    private Long employeeId;
```

Comment:
Why it removed?

Reply:
Because, bookingbasedto has also employeeId, I deleted dublicate code.